jsonnet/kube-prometheus/components/node-exporter.libsonnet

```diff
@@ -45,6 +45,13 @@ local defaults = {
       runbookURLPattern: 'https://runbooks.prometheus-operator.dev/runbooks/node/%s',
     },
   },
+  // NOTE: ignore veth network interface associated with containers.
+  // OVN renames veth.* to <rand-hex>@if<X> where X is /sys/class/net/<if>/ifindex
+  // thus [a-z0-9] regex below
+  ignoredNetworkDevices:: [
```

Comment:
maybe move next to `filesystemMountPointsExclude` and have a string rather than an array for consistency?